ansible_runner/utils/capacity.py

```diff
@@ -0,0 +1,17 @@
+import multiprocessing
+import resource
+
+
+def get_cpu_capacity():
```

Comment:
We are talking about having this return "raw data", as opposed to the `capacity` number as it will appear in AWX `/api/v2/instances/`

Since that's the case, I would prefer to use a method name other than `get_cpu_capacity`. Call it `get_cpu_count` or something like that to make it more clear. That will also help to allow us to import this freely in the AWX code.